OpenOversight/tests/routes/test_auth.py

```diff
@@ -465,15 +467,15 @@ def test_disabled_user_cannot_visit_pages_requiring_auth(client, session):
     with current_app.test_request_context():
         # Temporarily enable account for login
         user = User.query.filter_by(email=MOD_DISABLED_USER_EMAIL).one()
-        user.is_disabled = False
+        user.disabled_at = None
+        user.disabled_by = None
         session.add(user)
 
         rv, _ = login_modified_disabled_user(client)
         assert b"/user/sam" in rv.data
 
         # Disable account again and check that login_required redirects user correctly
-        user.is_disabled = True
-        session.add(user)
+        user.disable_user(User.query.filter_by(is_administrator=True).first().id)
```

Comment:
why are we doing `is_admin` here?

Reply:
The way the `disable_user` function is setup, it requires a `user.id` so that we can see who made the action. In this case, I just pulled the first administrator user I could find and took that ID.

Reply:
That's very helpful and makes perfect sense. Thanks!